src/app/self-service/users/user.service.ts

```diff
@@ -54,4 +54,14 @@ export class UserService {
     return this.http.disableApiPrefix().get('/assets/mock/user.mock.json');
   }
 
+  /**
+   * Change User Password
+   * @param userId User Id of users
+   * @param password New Password of the user
+   * @returns {Observable<any>}
+   */
+  changePassword(userId: string, passwordObj: any) {
```

Comment:
@BLasan just add a comment here `// TODO: update endpoint once API available`, then we'll merge

Reply:
Ok, I'll update that now

Reply:
Done. Please review